src/app.tsx

```diff
@@ -8,9 +8,14 @@ import { GlobalKeybindHandler } from "./keybind-handler.js";
 import { KeybindsDisplay } from "./keybinds-display.js";
 import { createFilePersister } from "./lib/query-storage-persister.js";
 
+const isDev = process.env.NODE_ENV === "development";
+
 const queryClient = new QueryClient({
   defaultOptions: {
     queries: {
+      refetchOnWindowFocus: !isDev,
+      refetchOnReconnect: !isDev,
+      staleTime: isDev ? 1000 * 60 * 5 : undefined,
```

Comment:
Might have to up this amount, unsure whether it'll automatically refresh every 5 minutes